ubuntu-troubleshoot.sh

```diff
@@ -0,0 +1,269 @@
+#!/bin/bash
+
+# copy the the following content into troubleshoot.sh file
+# and execute chmod +x ubuntu-troubleshoot.sh
+#
+# run this script with the absolute path of mount point.
+# sudo ./troubleshoot.sh <mountpoint> <server-ip> 
+# parameter 1 => absolute path of mount point
+# parameter 2 => server ip address
+
+# To-do:
+# - Add colors to the output
+# - Add support for aznfs
+# - Capture only limited packets in tshark
+
+# Need super user access to certain commands
+if [ "$(id -u)" -ne 0 ]; then
+    echo "Run this script as root!"
+    exit 1
+fi
+
+# Check for the number of parameters to the script
+if [[ ($# != 2) || ("$1" == "") || ("$2" == "") ]]; then
+    echo "usage: $0 <absolute mount path> <server ip>"
+    exit 1
+fi
+
+randomnum=$$
```

Comment:
use common "date+time encoded" extension for both the log file as well as any temp data files the run produces, for easier linking
202404151023